### ISSUE TITLE
Voip kit is not working in terminated state

### ISSUE THREAD

**Issue by prashantJangidSys** [ConnectyCube/connectycube-flutter-samples#308]
**Specify the sample to which the issue belongs (use [x]):**
[] Chat sample
[] P2P Calls sample
[x] Conference Calls sample

**Platform (use [x])**
[x] Android
[] iOS
[] macOS
[] Windows
[] Web
[] Linux

**Device info**
Manufacture: 
Model: 
OS version:

**Describe the bug:*
 - VoIP kit is working in foreground and background mode but not working in terminated state.*
Is there something which I am missing for terminated state?
- As accept the call from Incoming Call screen , the calling notification is not dismissed .How can i resolve this?

**Comment by TatankaConCube** [comment]
Can you reproduce this issue in our P2P Calls sample? Don't you receive any notifications in the terminated state? Or you can't accept/reject the call from the terminated state?

**Comment by prashantJangidSys** [comment]
I have done this in P2P call but this isn't working in conference call in terminated state only.
I am not getting notification in terminated state.

**Comment by prashantJangidSys** [comment]
and one more issue , we are initiating the conference call from web app but on ending the call,getting the error
-CB-SDK: ConferenceSession: Trying to leave from room, while session has been already closed
Error: Unexpected null value.

but this is working fine when executed in apk.

**Comment by TatankaConCube** [comment]
> -CB-SDK: ConferenceSession: Trying to leave from room, while session has been already closed
Error: Unexpected null value.

could you please provide the full log from the console log (from starting the call till getting this issue)?

**Comment by TatankaConCube** [comment]
> I am not getting notification in terminated state.

please try to use the `logcat` util for listening to the logs on the native Android side, try to clean the log before starting the call, and collect all logs after starting the call. there you should see something related to FCM. please collect everything in 'logcat' and share it here. Additionally please specify the device model, OS version, and other helpful info about your Android device

**Comment by Badar25** [comment]
Same for me, Its working in foreground state but not in background & terminated state for iOS. Any fix?

**Comment by TatankaConCube** [comment]
no activity for a long time. closing... please create a new ticket with the required info if the issue still actual
